FlightReservations/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+import {Login} from '../../screens/Login/Login';
+import {MyFlight} from '../../screens/MyFlight/MyFlight';
+import {Booking} from '../../screens/Booking/Booking';
+import {Register} from '../../screens/Register/Register';
+
+const Stack = createStackNavigator();
+
+function Navigation() {
+  return (
+    <Stack.Navigator>
+      <Stack.Screen
+        name="Register"
+        component={Register}
+        // options={{headerShown: false}}
+      />
+      <Stack.Screen name="Login" component={Login} options={{title: ''}} />
```

Comment:
You can add directly here the options={{headerShown: false}} to avoid rewriting it on every screen :)